app/assets/stylesheets/responsive/_global_style.scss

```diff
@@ -319,14 +319,29 @@ div.pagination {
     &:focus {
       background-color: transparent;
     }
+  }
+  a {
+    text-decoration: none;
+    color: #333;
+    &:hover,
+    &:active,
+    &:focus {
+      text-decoration: underline;
+    }
+  }
+  .action-menu__info-link {
     a {
-      text-decoration: none;
-      color: #333;
-      &:hover,
-      &:active,
-      &:focus {
-        text-decoration: underline;
-      }
+      font-size: 0.625em; //10px
```

Comment:
this makes the "Help" text uncomfortably small for me 😢 

![screen shot 2016-10-18 at 15 07 06](https://cloud.githubusercontent.com/assets/27760/19941375/c8c2c238-a127-11e6-966f-ddf7cad4d9f8.png)

Is that what 10px looks like? 🔍